webapp/channels/src/components/leave_team_modal/leave_team_modal.tsx

```diff
@@ -83,66 +81,70 @@ export default class LeaveTeamModal extends React.PureComponent<Props, State> {
         if (isGuest) {
             if (numOfPublicChannels !== 0 && numOfPrivateChannels !== 0) {
                 modalMessage = (
-                    <FormattedMarkdownMessage
-                        id='leave_team_modal_guest.desc'
-                        defaultMessage="** You will be removed from {num_of_public_channels} public { num_of_public_channels,plural,one {channel} other {channels}} and {num_of_private_channels} private { num_of_private_channels,plural,one {channel} other {channels}} on this team.** You won't be able to rejoin it without an invitation from another team member. Are you sure?"
+                    <FormattedMessage
+                        id='leave_team_modal_guest.description'
+                        defaultMessage="<strong>You will be removed from {num_of_public_channels}</strong> public {num_of_public_channels,plural,one {channel} other {channels}} and <strong>{num_of_private_channels}</strong> private {num_of_private_channels,plural,one {channel} other {channels}} on this team. You won't be able to rejoin it without an invitation from another team member. Are you sure?"
                         values={{
                             num_of_public_channels: numOfPublicChannels,
                             num_of_private_channels: numOfPrivateChannels,
+                            strong: (chunks: React.ReactNode) => <strong>{chunks}</strong>,
                         }}
                     />
                 );
             } else if (numOfPublicChannels === 0) {
                 modalMessage = (
-                    <FormattedMarkdownMessage
-                        id='leave_team_modal_guest_only_private.desc'
-                        defaultMessage="** You will be removed from {num_of_private_channels} private { num_of_private_channels,plural,one {channel} other {channels}} on this team.** You won't be able to rejoin it without an invitation from another team member. Are you sure?"
+                    <FormattedMessage
+                        id='leave_team_modal_guest_only_private.description'
+                        defaultMessage="<strong>You will be removed from {num_of_private_channels}</strong> private {num_of_private_channels,plural,one {channel} other {channels}} on this team. You won't be able to rejoin it without an invitation from another team member. Are you sure?"
```

Comment:
```suggestion
                        defaultMessage="<strong>You will be removed from {num_of_private_channels} private {num_of_private_channels,plural,one {channel} other {channels}} on this team.</strong> You won't be able to rejoin it without an invitation from another team member. Are you sure?"
```

I guess this is how it was previously, Can you check others as well

See previously
`** You will be removed from {num_of_private_channels} private { num_of_private_channels,plural,one {channel} other {channels}} on this team.** `